pandas/core/arrays/floating.py

```diff
@@ -410,6 +410,14 @@ def max(self, *, skipna=True, **kwargs):
         nv.validate_max((), kwargs)
         return super()._reduce("max", skipna=skipna)
 
+    def round(self, decimals=0):
```

Comment:
should do this generically (and move to NumericArray), e.g. something like

```
def round(self, decimals=0):
     return self._apply(np.round, decimals=decimals)
```

where _apply handles the masking & recasting. I think we discussed similar in `.to_numeric`

Reply:
or event better

```
round = _apply_function(np.round, decimals=0, "nice doc-string here")
```

Reply:
If we're going with the second way (may as well) would _apply_function be method on NumericArray (versus a module level function)?